src/version.rc

```diff
@@ -23,12 +23,12 @@ BEGIN
         BEGIN
             VALUE "CompanyName", "\0"
             VALUE "FileDescription", "SDL_sound\0"
-            VALUE "FileVersion", "2, 0, 3, 0\0"
+            VALUE "FileVersion", "3, 0, 0, 0\0"
             VALUE "InternalName", "SDL_sound\0"
             VALUE "LegalCopyright", "Copyright � 2023 Ryan C. Gordon\0"
```

Comment:
We use `(C)` in the SDL satellite libraries to not have issues with encodings.
Also, we still have one week left in 2024 ;)
```suggestion
            VALUE "LegalCopyright", "Copyright (C) 2024 Ryan C. Gordon\0"
```

Reply:
done ..